src/Relude/Numeric.hs

```diff
@@ -0,0 +1,76 @@
+{-# LANGUAGE RankNTypes          #-}
+{-# LANGUAGE ScopedTypeVariables #-}
+{-# LANGUAGE TypeApplications    #-}
+
+{- |
+Copyright: (c) 2018 Kowainik
```

Comment:
There was a copy-paste error in my initial code... Let's align properly in new modules 🙂 
```suggestion
Copyright:  (c) 2018 Kowainik
```